src/main/java/de/unijena/cheminf/mortar/controller/HistogramViewController.java

```diff
@@ -1001,7 +961,15 @@ private StackPane createStackPaneWithContextMenuAndStructureDisplayForBar(ImageV
                     true,
                     true);
             anImageView.setImage(tmpImage);
-        });
+        };
+        /* Event to open context menu (right click) to copy SMILES string or structure.
+           Context menu also opens, if  a right click on the frequency label is detected.
+         */
+        EventHandler<ContextMenuEvent> tmpContextMenuEvent = event -> {
```

Comment:
Minor comment on naming, this is an event *handler*, not an event (also above the "tmpMouseHoverEvent"). Apart from that, very nicely simplified, well done!